torch/autograd/variable.py

```diff
@@ -766,6 +766,12 @@ def multinomial(self, num_samples=1, with_replacement=False):
     def bernoulli(self):
         return Bernoulli()(self)
 
+    def zero(self):
+        return Zero.apply(self, False)
```

Comment:
There's no `Tensor.zero`. We plan to unify both types in the future, so let's not diverge their interfaces any more.

Reply:
@apaszke - ah just commented regarding this below. I'll remove this then